.github/workflows/zephyr_repack_plugin_e2e_results.yml

```diff
@@ -0,0 +1,55 @@
+name: Process Zephyr Repack Plugin E2E Test Results
+
+on:
+  workflow_dispatch:
+    inputs:
+      workflow_status:
+        description: "Status of the overall E2E tests workflow"
+        required: true
+      run_url:
+        description: "URL to the workflow run"
+        required: true
+      zephyr_deployment_ios_status:
+        description: "Status of zephyr-deployment-ios"
+        required: true
+      ios_tests_status:
+        description: "Status of ios-tests"
+        required: true
+      zephyr_deployment_android_status:
+        description: "Status of zephyr-deployment-android"
+        required: true
+      android_tests_status:
+        description: "Status of android-tests"
```

Comment:
Just to be sure - we need those urls to report back the status, logs and all the necessary stuff back to this repository, right?

Reply:
Right you are 

Here's how this urls looks like:
![image](https://github.com/user-attachments/assets/a1e9cdd0-c86e-4d1b-9ee3-f488958febcd)

That's basically the links to the GH workflow run and its exact job logs